gadget.yaml

```diff
@@ -29,9 +29,17 @@ volumes:
           edition: 2
         content:
           - source: grubx64.efi
-            target: EFI/boot/grubx64.efi
+            target: EFI/ubuntu/grubx64.efi
           - source: shim.efi.signed
-            target: EFI/boot/bootx64.efi
+            target: EFI/BOOT/BOOTX64.efi
+          - source: fbx64.efi
+            target: EFI/BOOT/fbx64.efi
+          - source: mmx64.efi
+            target: EFI/BOOT/mmx64.efi
```

Comment:
Do we really want MokManager in the gadget? @anonymouse64 @pedronis @mvo5

Reply:
I think there were requests for `mokutil` in core before, and also requests to thus ship mokmanager in the gadget. But I cannot find that any more. Maybe ping Field? CE? @vorlonofportland do you recall any requests or need for mokmanager on Core?